### DISCUSSION TITLE
Perpetual license for Unity 2023

### DISCUSSION THREAD

**Discussion by atmuc** [visualdesigncafe/nature-renderer#42] [Help]
In Unity asset store page I get this info  "Nature Renderer 2023 - Perpetual license for Unity 2023 (Coming Soon)"

If I buy Perpetual license for Unity 2022 will it be free to upgrade to 2023?

Is Nature Renderer compatible with Gaia?

**Comment by stonstad** [comment]
Did you find an answer to your question?  I asked the same thing... radio silence.
